plumbing/format/packfile/scanner.go

```diff
@@ -275,19 +276,28 @@ func (s *Scanner) NextObject(w io.Writer) (written int64, crc32 uint32, err erro
 // ReadRegularObject reads and write a non-deltified object
 // from it zlib stream in an object entry in the packfile.
 func (s *Scanner) copyObject(w io.Writer) (int64, error) {
```

Comment:
I think this change is not related to this PR, Could fix this naming issue into another PR?